### DISCUSSION TITLE
33.1.1.533_BETA SDK compiling aab language errors

### DISCUSSION THREAD

**Discussion by mehmury** [airsdk/Adobe-Runtime-Support#926] [Q&A] [answered]
I tried to get android aab output with Adobe air HARMAN 33.1.1.533_BETA SDK. After several trials and errors, the last error I got was unable to define the descriptions and names set in different languages in the main xml description file and it keeps giving this error.

ERROR'S on cmd:
application.description.text@xml:lang contains an invalid value
application.name.text@xml:lang contains an invalid value

I get the same error when compiling apk with Animate.(33.1.1.533_beta)
should i wait for real version of sdk?

**Comment by ajwfrost** [comment, marked as answer]
Yes, sorry, the xml property validation wasn’t working properly in the beta but that’s working again in the proper release …

thanks

**Reply by mehmury** [reply]
ok...thanks
